### ISSUE TITLE
Wrong message when attempting to Mute the master volume

### ISSUE THREAD

**Issue by MichalSuchy** [yplassiard/nvda-soundmanager#2]
Muting master volume doesn't work. Nvda says: "Unable to retrieve current application". It can be useful, when computer or notebook hasn't button for quick muting master volume.

**Comment by yplassiard** [comment]
Hi,
This is currently not possible due to a limitation of the underlying the add-on uses to control volumes.
The message is, however, irrelevant and will be fixed, thanks

**Comment by yplassiard** [comment]
### Accepted

This is true and has been fixed in 2019.06.1, which will be out soon.
